app/Http/Controllers/Admin/RoomTypeController.php

```diff
@@ -94,13 +98,19 @@ public function edit($id)
      */
     public function update(RoomTypeUpdateFormRequest $request, $id)
     {
+        $columns = [
+            'id',
+            'name',
+            'quality',
+            'image',
+        ];
         $updateInfo = $request->only('name', 'quality');
         try {
-            $roomType = RoomType::findOrFail($id);
+            $roomType = RoomType::findOrFail($id, $columns);
             $oldImage = $roomType['image'];
             if ($request->hasFile('image')) {
                 $updateInfo['image'] = $this->imageUpload($this::UPLOAD_KEY, $request->file('image'));
-                $this->imageRemove('roomtype', $oldImage);
+                $this->imageRemove($this::UPLOAD_KEY, $oldImage);
```

Comment:
I think error here, pls check. `$this::UPLOAD_KEY` is remove from this controller